kore-rpc-types/src/Kore/JsonRpc/Server.hs

```diff
@@ -76,31 +75,32 @@ runJSONRPCT encodeOpts ver ignore snk src f = do
 
 -- | TCP server transport for JSON-RPC.
 jsonRpcServer ::
-    (MonadLoggerIO m, MonadUnliftIO m, FromRequestCancellable q, ToJSON r) =>
+    (MonadUnliftIO m, FromRequestCancellable q, ToJSON r) =>
     -- | Connection settings
     ServerSettings ->
     -- | Action to perform on connecting client thread
-    (Request -> Respond q (Log.LoggingT IO) r) ->
+    (Request -> Respond q IO r) ->
     [JsonRpcHandler] ->
     m a
 jsonRpcServer serverSettings respond handlers =
     runGeneralTCPServer serverSettings $ \cl ->
-        runJSONRPCT
-            -- we have to ensure that the returned messages contain no newlines
-            -- inside the message and have a trailing newline, which is used as the delimiter
-            rpcJsonConfig{Json.confIndent = Spaces 0, Json.confTrailingNewline = True}
-            V2
-            False
-            (appSink cl)
-            (appSource cl)
-            (srv respond handlers)
-
-data JsonRpcHandler = forall e. Exception e => JsonRpcHandler (e -> Log.LoggingT IO ErrorObj)
+        Log.runNoLoggingT $
```

Comment:
we simply disable the internal JSONRPC library logging here. I don't think the output was ever useful past initial debugging of the server implementation.